File: data/example/code.java
import java.io.*;
public class code {
	public static void main(String args[])throws IOException{
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
		int n;
		String str;
		while((str=in.readLine())!=null){
			n = Integer.parseInt(str);
			n = n*n;
			System.out.println(n);
			} // while
		}
	}

File: data/example/code.js
importPackage(java.io);
importPackage(java.lang);
var reader = new BufferedReader( new InputStreamReader(System['in']) );
while (true){
    var line = reader.readLine();
    if (line==null) break;
    else {
        i = parseInt(line);
        System.out.println((i*i)+'');
        }
    }